libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/getMappedConfigForWidget.ts

```diff
@@ -34,12 +38,24 @@ const localIdOrDie = (ref: ObjRefInScope): string => {
     throw new Error("invalid invariant");
 };
 
-const getTitleFromDrillableItemPushData = (items: IAvailableDrillTargets, measureId: string): string => {
+const getTitleFromDrillableItemPushData = (items: IAvailableDrillTargets, itemId: string): string => {
     const measureItems = items.measures || [];
-    const res = measureItems.find(
-        (x: IAvailableDrillTargetMeasure) => x.measure.measureHeaderItem.localIdentifier === measureId,
+    const measureResult = measureItems.find(
+        (measureResult: IAvailableDrillTargetMeasure) =>
+            measureResult.measure.measureHeaderItem.localIdentifier === itemId,
     );
-    return res ? res.measure.measureHeaderItem.name : "";
+
+    if (!measureResult) {
+        const attributeItems = items.attributes || [];
+        const attributeResult = attributeItems.find(
+            (attributeItem: IAvailableDrillTargetAttribute) =>
+                attributeItem.attribute.attributeHeader.localIdentifier === itemId,
+        );
+
+        return attributeResult ? attributeResult.attribute.attributeHeader.formOf.name : "";
+    }
+
+    return measureResult ? measureResult.measure.measureHeaderItem.name : "";
```

Comment:
Detail: You do not need to check `measureResult` here, it being not defined is handled by the if block above.